src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthComments.java

```diff
@@ -1,26 +1,33 @@
 package com.puppycrawl.tools.checkstyle.checks.sizes.methodlength;
 
-import com.puppycrawl.tools.checkstyle.api.DetailAST;
-import com.puppycrawl.tools.checkstyle.api.TokenTypes;
-
 public class InputMethodLengthComments {
-    public void visitToken(DetailAST ast) {
+    static class DetailClass {
```

Comment:
pitest requires these classes to be at the start because of a mutation that only occurs near the end of the file.